services/api-server/src/simcore_service_api_server/services_http/catalog.py

```diff
@@ -64,6 +65,21 @@ def to_solver(self) -> Solver:
             **data,
         )
 
+    def to_program(self) -> Program:
+        data = self.model_dump(
+            include={"name", "key", "version", "description", "contact", "owner"},
+        )
+        return Program(
+            id=data.pop("key"),
```

Comment:
TIP: check mapping utils we have in packages/common-library/src/common_library/dict_tools.py specifically
`remap_keys`. Do a search to see usage